index.md

```diff
@@ -84,9 +111,17 @@ This pipeline will produce a number of directories and files:
 - **multisample.vcf.gz**: A multisample VCF file with the variantcalling
 results.
 
-## About
-This pipeline is part of [BioWDL](https://biowdl.github.io/)
-developed by [the SASC team](http://sasc.lumc.nl/).
+## Scattering
+The pipeline performs scattering to speed up analysis on grid computing
+clusters. This means that the reference genome will be split into regions of
+roughly equal size (see the `scatterSize` inputs). For certain steps each of
+these regions will be analyzed in separate jobs, allowing them to be processed
+in parallel.
+
+## Tool versions
```

Comment:
I would put this Tool versions section way before, e.g., right after Usage section. And change the section head to "Dependency requirement and tool versions"